diff -rup zip-1.2.0-orig/pyproject.toml zip-1.2.0/pyproject.toml
--- zip-1.2.0-orig/pyproject.toml	2021-03-23 10:55:37.000000000 +0100
+++ zip-1.2.0/pyproject.toml	2021-03-23 10:56:05.000000000 +0100
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools>=34.4", "wheel", "setuptools_scm>=1.15"]
+requires = ["setuptools<42.0", "wheel", "setuptools_scm<6.0.0"]
 build-backend = "setuptools.build_meta"

 [tool.black]
